slitlessutils/core/preprocess/astrometry/downgrade_wcs.py

```diff
@@ -1,19 +1,96 @@
-import os
-
 from astropy.io import fits
 
 from ....logger import LOGGER
+from . import utils
+
+
+def _downgrade_wcs_one(imgfile, key, mode, newfile, inplace):
```

Comment:
Just curious, but why the "_one" in this function name?

Reply:
Yeah the function with the "one" suffix is a "worker", I guess?  The main function called the worker routine in a loop.  I did this because I thought it would be cleaner in juggling all the if/then conditions.

Reply:
I think it's fine to just call this `_downgrade_wcs`, the underscore is enough to differentiate the private method that gets called by the public one.

Reply:
Ok.. sounds good.